modules/dxp/apps/portal-workflow/portal-workflow-metrics-rest-impl/src/main/java/com/liferay/portal/workflow/metrics/rest/internal/odata/entity/v1_0/InstanceEntityModel.java

```diff
@@ -30,6 +31,7 @@
 
 	public InstanceEntityModel() {
 		_entityFieldsMap = Stream.of(
+			new StringEntityField("assetType", locale -> "assetType"),
```

Comment:
Hey @MarinhoFeliphe, as it is a internationalized field use `new StringEntityField(
				"assetType", locale -> Field.getSortableFieldName("assetType"))` instead. Thx

Reply:
just fixing an error in the proposed solution, use `new StringEntityField(
				"assetType",
				locale -> Field.getSortableFieldName(
					"localized_assetType_" + LocaleUtil.toLanguageId(locale))));`